docs/installation-and-operations/installation/docker/README.md

```diff
@@ -173,6 +181,23 @@ docker run -d -e KEY1=VALUE1 -e KEY2=VALUE2 ...
 docker run -d --env-file path/to/file ...
 ```
 
+#### Disabling HTTPS mode
+
+By default, OpenProject will expect a HTTPS request in production systems.
+In most cases, you will have an external web server or load balancer terminating the SSL/TLS connection and proxy/reverse-proxy to the docker container. You will then have to set up the web server to forward the protocol information (usually, this is X-Forwarded-Proto but depends on your web server).
+
+**Note**: This does not imply the docker container itself is running on SSL.
+
+If you _really_ want to disable HTTPS responses by OpenProject, you will need to add the environment variable `OPENPROJECT_HTTPS=false`. Note that this will disable secure cookies for session cookies, and is strongly discouraged for any production system.
+
+#### Disabling HSTS headers and insecure request upgrade
+
+If OpenProject is running as HTTPS, it will output HTTP Strict Transport Security (HSTS) headers by default, which will force the browser to upgrade a request to HTTPS even before calling the web server. On the server side, if a request is still unsecured, OpenProject will return a redirect to HTTPS.
```

Comment:
```suggestion
If OpenProject is running as HTTPS, it will output HTTP Strict Transport Security (HSTS) headers by default, which will force the browser to upgrade a request to HTTPS even before calling the web server. On the server side, if a request is still issued through HTTP, OpenProject will return a redirect to HTTPS.
```

Precise what `unsecured` means